README.md

```diff
@@ -1,6 +1,7 @@
 # Amazon Chime SDK for JavaScript
 [Amazon Chime SDK React Components](https://github.com/aws/amazon-chime-sdk-component-library-react)
-[Project Board](https://aws.github.io/amazon-chime-sdk-js/modules/projectboard.html)
+
+[Amazon Chime SDK Project Board](https://aws.github.io/amazon-chime-sdk-js/modules/projectboard.html)
```

Comment:
How about moving this above the React components link since this is the project board?

Reply:
sounds good